streamlit_web.py

```diff
@@ -380,15 +385,57 @@ def sentiment():
         )
         question_senti(senti_df)
 
+def senti_pos_iter(tokens_column):
+    # Start off with an empty list
+    display_series = []
+    # For each item in the already existing tokens column in the data frame,
```

Comment:
I'm hesitant about whether to move this function and `senti_neg_iter` to `analyzer.py`. The `streamlit_web.py` is complex enough, and since these two functions are more encapsulated, probably a better idea to move to another module. 

Also, is it possible to find a better abstract for these two functions? There are many repetitive codes here.

Also, once this is moved to another module, I would like your team to provide tests for it if possible.